entry.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * entry.js
+ * Simple wrapper for babel
+ */
+
+require('babel-register')
+require('babel-polyfill')
+
+var currentIndex = process.argv.indexOf(__filename)
```

Comment:
It's a generic script for babel polyfill that I made. So you would do something like `node entry.js master.js` and tada, all of babel. Currently use it for the slave and master script. Keeps everything nice and fresh looking so we don't have server scripts being in regular ol boring javascript.